terraform/modules/benchmark/service/benchmark_base.rb

```diff
@@ -166,20 +187,16 @@ def run_validation_step(daily, args, metrics)
 
   # Import snapshot, write metrics, and call validation function, returning metrics.
   def import_and_validation(daily, args, metrics)
-    import_command = splice_args(@import_command, args)
-    metrics[:import] = exec_command(import_command)
+    handle_exception(true) do
```

Comment:
I do understand the nested call to `handle_exception` but maybe we could force deletion of `db` in all cases?

Reply:
Just documenting our latest in person discussion: the `--force` flag is already specified, but the `-c` flag fails if `handle_exception` is called before the `data_dir` is created, so we need to keep the nested call with a `db_created` argument to specify when the `data_dir` is created.

Reply:
Ok then maybe we could just test the presence of the config file, and if present call the `db destroy` whether the db is existing or not (if db is not found, the command still returns a success).